README.md

```diff
@@ -48,6 +48,14 @@ var db = marklogic.createDatabaseClient({
   authType: 'DIGEST'
 });
 
+// For MarkLogic Cloud
+let db = marklogic.createDatabaseClient({
```

Comment:
Small thing, but both this and the above example should do `const db` as there will rarely ever be a reason to reassign the variable to something else, and thus `const` is a good practice to encourage. 

Also, I think a more realistic value for "basePath" would be useful here - something like "/ml/manage/test", or you can copy what's at https://github.com/marklogic/ml-gradle/wiki/MarkLogic-Cloud-support .